src/code/z_collision_check.c

```diff
@@ -1271,6 +1250,10 @@ ColChkResetFunc sOCResetFuncs[] = {
     Collider_ResetQuadOC,   Collider_ResetSphereOC,
 };
 
+TriNorm D_801EE6C8;
+u8 D_801EE6FC_unused[4];
```

Comment:
```suggestion
static s32 sBssPad;
```
What we have been calling unused bss variables. Though kind of weird this is needed here, but not with the other TriNorm pairs but if it works it works I guess🤷 .

Reply:
edited